setup.py

```diff
@@ -21,14 +21,15 @@
 long_description = (cwd / "README.md").read_text()
 
 SPECIAL_DEPENDENCIES_FOR_RELEASE = [
-    "idaes-pse>=2.5.0,<2.6.0rc0",  # from PyPI
+    # "idaes-pse>=2.5.0,<2.6.0rc0",  # from PyPI
+    "idaes-pse @ https://github.com/IDAES/idaes-pse/archive/refs/heads/dynamic_cv0d.zip",
```

Comment:
This can (and should) be reverted since `SPECIAL_DEPENDENCIES_FOR_RELEASE` is only used for releases